dpnp/backend/include/dpnp_gen_2arg_3type_tbl.hpp

```diff
@@ -138,16 +138,16 @@ MACRO_2ARG_3TYPES_OP(dpnp_divide_c,
                                         std::complex<double>))
 
 MACRO_2ARG_3TYPES_OP(dpnp_fmod_c,
-                     sycl::fmod((double)input1_elem, (double)input2_elem),
-                     nullptr,
-                     std::false_type,
+                     dispatch_fmod_op(input1_elem, input2_elem),
+                     x1 % x2,
```

Comment:
can be the `dispatch_fmod_op` used in vector implementation also?
```suggestion
                     dispatch_fmod_op(x1, x2),
```

Reply:
Yes, sure!
Done